### ISSUE TITLE
Tiny MCE Table Format Issue

### ISSUE THREAD

**Issue by nowfal18** [tinymce/tinymce#6106]
I am trying to copy some contents from microsoft excel to the text editor.The coloring and format is not working.I am using windows mechine.But Linux and Safari same code working.
 tinymce.init({
            selector: "#ReqDesc",
            menubar: false,           
            height: 400,
            width: "100%",            
            plugins: [
                "advlist autolink lists charmap print preview hr anchor pagebreak",
                "searchreplace visualblocks visualchars code insertdatetime media nonbreaking",
                "save table contextmenu directionality paste textcolor"
            ],
            toolbar: 'fontselect | fontsizeselect | bold italic underline forecolor | alignleft aligncenter alignright alignjustify | outdent indent | code',
            toolbar_location: 'bottom',
            skin: 'outside',           
            paste_webkit_styles: 'all',
            paste_retain_style_properties: 'all',                  
            table_default_attributes: {
                border: '1'
            },

**Comment by lincolndennis** [comment]
Can you be a little more specific in what you mean by "coloring and format is not working"? In what way aren't they working? Are they not being retained at all when pasted, or are they not rendering as you'd except in the editor, or something else? Can you confirm what version of TinyMCE you are using? Can you also confirm what specific browsers (name and version) you are testing with?

**Comment by nowfal18** [comment]
Dear Team,

I am working on a web project. I have copied some rows from Microsoft excel to the TinyMCE editor. But they not being retained at all when pasted, Means, in the input excel rows have background color and text color. When I paste it, all the colors and border are gone. When I check with TinyMCE website ,there is one sample control available .I paste the same input there, it's asking one option keep formatting, and when I have clicked on Keep Formatting, Its working well, otherwise not. But in my project those options not coming as well as not retaining the styles.

I was tested in Google Chrome Windows Version.
Version 74.0.3729.108 (Official Build) (64-bit)

TinyMCE used as free open source  control,not paid one.

Following are my Input

Get All Details of IOM Request By Passing request ID
Search for any attachment is available or not
If attachment available,take the GUID from URL to upload new attachment to one drive
If no attachment available ,then create new GUID to upload attachment to One Drive
Call Document Upload API to Upload Document to One Drive and save the return URL to local.In Web we are saving in datatable.So its easy to delete the attachement from there if user want to remove .
If user make any action like approve,reject,rfi etc,then call the API for the file save .This API call after any action not while attach any document.


Thanks, and Regards
Nowfal Majeed





________________________________
